serving/ingress/pkg/reconciler/ingress/ingress_test.go

```diff
@@ -74,7 +74,7 @@ func TestKourierReconcile(t *testing.T) {
 		WantDeletes: []clientgotesting.DeleteActionImpl{{
 			ActionImpl: clientgotesting.ActionImpl{
 				Namespace: ingressNamespace,
-				Resource:  routev1.SchemeGroupVersion.WithResource("routes"),
+				Resource:  routev1.GroupVersion.WithResource("routes"),
```

Comment:
Curious: Why is this better than the other?

Reply:
@markusthoemmes it was deprecated.